<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_connect</class>
 <widget class="QDialog" name="Dialog_connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings connect</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="qbl_one">
       <item>
        <widget class="QLabel" name="l_HostName">
         <property name="text">
          <string>&amp;Host Name</string>
         </property>
         <property name="buddy">
          <cstring>le_HosName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_HosName">
         <property name="text">
          <string>localhost</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_Port">
         <property name="text">
          <string>&amp;Port</string>
         </property>
         <property name="buddy">
          <cstring>sb_Port</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_Port">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32000</number>
         </property>
         <property name="value">
          <number>5432</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="l_BDName">
       <property name="text">
        <string>&amp;Data Base Name</string>
       </property>
       <property name="buddy">
        <cstring>le_BDName</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_BDName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="l_Login">
       <property name="text">
        <string>&amp;Login</string>
       </property>
       <property name="buddy">
        <cstring>le_Login</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_Login"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="l_Password">
       <property name="text">
        <string>&amp;Password</string>
       </property>
       <property name="buddy">
        <cstring>le_Password</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_Password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pb_d_OK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pb_d_Cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_HosName</tabstop>
  <tabstop>sb_Port</tabstop>
  <tabstop>le_BDName</tabstop>
  <tabstop>le_Login</tabstop>
  <tabstop>le_Password</tabstop>
  <tabstop>pb_d_OK</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pb_d_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_connect</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
